boards/arm/lpcxpresso54114/lpcxpresso54114.dtsi

```diff
@@ -20,12 +13,6 @@
 		led2 = &blue_led;
 	};
 
-	chosen {
-		zephyr,sram = &sram0;
-		zephyr,flash = &flash0;
-		zephyr,console = &usart0;
-	};
-
 	leds {
```

Comment:
Having the LEDs shared between both boards seems problematic.  Not sure if it makes sense to have all the LEDs on one core, or split them.

Reply:
It's the same problem as having a uart instance available to both cores on the soc. How about we handle the leds the same way, where we define them in the shared dtsi file with `status = "disabled"`, and then the core-specific dts can choose to enable them. This way we can enable them by default for the m4, but allow an override via a dts overlay if an application wants to use them in the m0 instead.

Reply:
That works, just didn't want the default case to have both M0 & M4 accessing them.

Reply:
Done